src/common/interfaces/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './put-options.interface';
 export * from './unprocessable-entity-error.interface';
 export * from './workos-options.interface';
 export * from './workos-response-error.interface';
+export * from './pagination-options.interface'
```

Comment:
You're going to want to make sure your code editor is setup to "Add new line at end of file" on save.

Reply:
awesome tip - done.